src/task 22/task22.js

```diff
@@ -0,0 +1,31 @@
+function buildTable(arr){
+    var Table = document.querySelector("#table");
+    var headers = Object.keys(arr[1]);
```

Comment:
You should have a more generic solution. What if every entry will have 2 unique property only? Resulted table should have all the meet properties in it for each entry (all not met per entry will be empty/blank/-).